.github/workflows/e2e.yml

```diff
@@ -0,0 +1,83 @@
+# Copyright © 2023 Cask Data, Inc.
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+# Note: Any changes to this workflow would be used only after merging into develop
+name: Build e2e tests
+
+on:
+  workflow_run:
+    workflows:
+      - Trigger build
+    types:
+      - completed
+
+jobs:
+  build:
+    runs-on: k8s-runner-e2e
+
+    if: ${{ github.event.workflow_run.conclusion != 'skipped' }}
+
+    steps:
+      # Pinned 1.0.0 version
+      - uses: haya14busa/action-workflow_run-status@967ed83efa565c257675ed70cfe5231f062ddd94
+      - uses: actions/checkout@v3
+        with:
+          path: plugin
+          ref: ${{ github.event.workflow_run.head_sha }}
+
+      - name: Checkout e2e test repo
+        uses: actions/checkout@v3
+        with:
+          repository: cdapio/cdap-e2e-tests
+          path: e2e
+
+      - name: Cache
+        uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ github.workflow }}-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-${{ github.workflow }}
+
+      - name: Run tests
+        run: python3 e2e/src/main/scripts/run_e2e_test.py
```

Comment:
Don't we need to add any secret values here for authentication to run tests as we do in another repo's?

Reply:
Right now we are executing the design time e2e tests on git actions, ideally not required for now.